index.js

```diff
@@ -0,0 +1,135 @@
+const exec = require('@actions/exec');
+const core = require('@actions/core');
+const github = require('@actions/github');
+const { parse: gitDiffParser } = require('what-the-diff');
+const fs = require('fs-extra');
+const simpleGit = require('simple-git');
+const env = process.env;
+
+function matchExact(r, str) {
+  let match = str.match(r);
+  return match && str === match[0];
+}
+
+function fixNewLineEOF(b) {
+  // If file is empty
+  if (b.length < 1) {
```

Comment:
Why is this check necessary?

Reply:
This is there to ignore empty files. I had needed it with a previous logic but it is no longer required with the current logic.